### ISSUE TITLE
Add more shards and apps

### ISSUE THREAD

**Issue by straight-shoota** [bcardiff/test-ecosystem#5]
I'd like to add more projects to improve the surface area of tests. We've already discovered two probable regressions in 1.1.0 from the invidious project. So I think that would be a good candidate to add here, for example.

Criteria to include projects here are: Because they are important for the Crystal ecosystem, a large code base (thus using many features) or using specific features of the compiler or standard library. And of course it should be reasonably easy to run test (ideally just checkout and run `make test`).

Having projects run their own specs on nightly and pre-release builds of Crystal is of course also valuable. I'd encourage every maintainer to do that (it's really easy on GHA with https://github.com/crystal-lang/install-crystal btw.)

Some proposals:

* [ ] https://github.com/iv-org/invidious/
* [x] https://github.com/athena-framework/athena
* [x] <del>https://github.com/ffwff/lilith</del>
* [ ] https://github.com/askn/crystal-by-example
* [x] https://github.com/crystal-community/crystal-patterns
* [ ] https://github.com/anykeyh/clear
* [ ] https://github.com/imdrasil/jennifer.cr
* [ ] https://github.com/spider-gazelle/spider-gazelle
* [ ] https://github.com/crystal-lang/shards
* [ ] https://github.com/crystal-lang/crystal-db
* [ ] https://github.com/crystal-lang/crystal-mysql
* [ ] https://github.com/crystal-lang/crystal-sqlite3
* [ ] https://github.com/will/crystal-pg
* [ ] https://github.com/stefanwille/crystal-redis
* [ ] https://github.com/luckyframework/lucky
* [ ] https://github.com/amberframework/amber

Please add more suggestions.

**Comment by beta-ziliani** [comment]
Well, we need to be sure that maintainers will want our PRs if we find an issue in their code. I mean, _some_ breakages might be good. I say this because I don't know how maintained are some of the listed projects.

**Comment by beta-ziliani** [comment]
We can always contact the maintainers in doubt. I'm not sure how active is Lilith, for instance, but I understand that this project might shake the language in a way few other projects might do.

**Comment by straight-shoota** [comment]
Yeah, I don't see the purpose of test-ecosystem to make sure these specific libraries work well with an upcoming release. For me it's primarily about identifying unintended breaking changes.

**Comment by beta-ziliani** [comment]
The thing is what we do if we break something _for a good reason_. We need to be able to tell the developers how to fix their code (and expect the library to progress, otherwise it has to be removed from the list).

**Comment by beta-ziliani** [comment]
Let remove lilith from the list, [it won't be updated](https://github.com/ffwff/lilith/issues/38) and [it doesn't compile now](https://github.com/ffwff/lilith/issues/37).
